.gitignore

```diff
@@ -21,3 +21,4 @@
 /sasmodels.egg-info/
 /example/Fit_*/
 /example/batch_fit.csv
+/sasmodels/models/lib/gauss*.c
```

Comment:
This will exclude the built-in 20, 76, and 150 point curves as well. This could lead to attempts to remove those files in the future.

Reply:
Done:
```
# Note: changes to gauss20, gauss76 and gauss150 are still tracked since they
# are part of the repo and required by some models.
```